### PR TITLE
Components: Sanckbars & Ratings

### PR DESCRIPTION
**Pull request by horsemaker**
* Added Snackbars
  * Baseline Snackbar
  * Leading Snackbar
  * Stacked Snackbar
* Added Ratings
  * Filled Star
  * Half Filled Star
  * Empty Star
  * 5-Star Ratings
  * 3-Star Ratings

### CODE REVIEW

File: components/ratings/index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          >neoG Camp</a
+        ></span
+      >
+    </footer>
+  </body>
+</html>
+
+<!-- 
+  <span class="material-icons">
+star
```
Comment (mimansha-swarup) [inline components/ratings/index.html]: remove unnecessary comments

**Review by mimansha-swarup — APPROVED** [review]
looks good to me

**Review by vijaytembugade — APPROVED** [review]
Amazing Stying. You can merge the changes.

File: components/ratings/index.html (hunk 1 above)
Comment (horsemaker) [reply components/ratings/index.html]: Forgot! Thanks!

**Review by t007rushi — APPROVED** [review]
Everything looks clean.

**Pull request merged**